airgun/entities/contenthost.py

```diff
@@ -141,6 +141,33 @@ def install_errata(self, entity_name, errata_id, install_via=None):
             view.wait_for_result()
         return view.read()
 
+    def install_all_errata(self, entity_name, install_via=None):
+        """Install all errata on a content host
+
+        :param name: content host name to apply errata on
+        :param str install_via: via which mean to install errata. Available
+            options: "katello", "rex", "rex_customize"
+
+        :return: Returns a dict containing task status details
+        """
+        view = self.navigate_to(self, 'Edit', entity_name=entity_name)
+        breakpoint()
```

Comment:
I suppose this isn't intended?

Reply:
oh dear

Reply:
@lhellebr Fixed now, thank you for spotting that.